stdlib/2/xmlrpclib.pyi

```diff
@@ -0,0 +1,193 @@
+    def __init__(self, target: _Unmarshaller) -> None: ...
+
+class Marshaller:
+    memo = ...  # type: MutableMapping[int, Any]
+    data = ...  # type: str
+    encoding = ...  # type: Optional[str]
+    allow_none = ...  # type: bool
+    def __init__(self, encoding: Optional[str] = ..., allow_none: bool = ...) -> None: ...
+    dispatch = ...  # type: Mapping[type, Callable[[Marshaller, str, Callable[[str], None]], None]]
+    def dumps(self, values: Iterable[str]) -> str: ...
+    def dump_nil(self, value: str, write: Callable[[str], None]) -> None: ...
+    def dump_int(self, value: str, write: Callable[[str], None]) -> None: ...
+    def dump_bool(self, value: str, write: Callable[[str], None]) -> None: ...
+    def dump_long(self, value: str, write: Callable[[str], None]) -> None: ...
+    def dump_double(self, value: str, write: Callable[[str], None]) -> None: ...
+    def dump_string(self, value: str, write: Callable[[str], None], escape: Callable[[AnyStr, Callable[[AnyStr, AnyStr, AnyStr], AnyStr]], AnyStr] = ...) -> None: ...
+    def dump_unicode(self, value: str, write: Callable[[str], None], escape: Callable[[AnyStr, Callable[[AnyStr, AnyStr, AnyStr], AnyStr]], AnyStr] = ...) -> None: ...
+    def dump_array(self, value: str, write: Callable[[str], None]) -> None: ...
+    def dump_struct(self, value: str, write: Callable[[str], None], escape: Callable[[AnyStr, Callable[[AnyStr, AnyStr, AnyStr], AnyStr]], AnyStr] = ...) -> None: ...
+    def dump_datetime(self, value: str, write: Callable[[str], None]) -> None: ...
+    def dump_instance(self, value: str, write: Callable[[str], None]) -> None: ...
+
+class Unmarshaller:
+    def append(self, object: Any) -> None: ...
+    def __init__(self, use_datetime: bool = ...) -> None: ...
+    def close(self) -> tuple: ...
+    def getmethodname(self) -> Optional[str]: ...
+    def xml(self, encoding: str, standalone: bool) -> None: ...
+    def start(self, tag: str, attrs: Any) -> None: ...
+    def data(self, text: str) -> None: ...
+    def end(self, tag: str, join: Callable[[Iterable[str], str], str] = ...) -> None: ...
+    def end_dispatch(self, tag: str, data: str) -> None: ...
+    dispatch = ...  # type: Mapping[str, Callable[[Unmarshaller, str], None]]
+    def end_nil(self, data: str): ...
+    def end_boolean(self, data: str) -> None: ...
+    def end_int(self, data: str) -> None: ...
+    def end_double(self, data: str) -> None: ...
+    def end_string(self, data: str) -> None: ...
+    def end_array(self, data: str) -> None: ...
+    def end_struct(self, data: str) -> None: ...
+    def end_base64(self, data: str) -> None: ...
+    def end_dateTime(self, data: str) -> None: ...
+    def end_value(self, data: str) -> None: ...
+    def end_params(self, data: str) -> None: ...
+    def end_fault(self, data: str) -> None: ...
+    def end_methodName(self, data: str) -> None: ...
+
+class _MultiCallMethod: ...
```

Comment:
Is there a reason `stubgen` (provided by Mypy) would generate stubs for it? Usually it avoids those. Though it *is* returned by MultiCall.__getattr__...